.storybook/preview.tsx

```diff
@@ -1,13 +1,14 @@
-import { Story } from '@storybook/react'
-import { IonApp } from '@ionic/react'
-import { setupIonicReact } from '@ionic/react'
-import { MemoryRouter } from 'react-router'
+import { IonApp, setupIonicReact } from '@ionic/react'
```

Comment:
I added eslinting to storybook files. These imports are being shuffled because of that.